rest-server/src/models/job.js

```diff
@@ -132,8 +132,7 @@ class Job {
       });
   }
 
-  putJob(name, data, username, next) {
-    const originData = data;
```

Comment:
just use `Object.assign` ?
```
putJob(name, originalData, ...) {
  const data = {};
  Object.assign(data, originalData);
  ...
}
```

Reply:
Actually this `originalData` you proposed is not so original, because it has been processed and altered by middlewares, thus we cannot guarantee that it can pass the input validation.